src/core/RecyclerListView.tsx

```diff
@@ -363,7 +369,13 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
                 onScroll={this._onScroll}
                 onSizeChanged={this._onSizeChanged}
                 contentHeight={this._initComplete ? this._virtualRenderer.getLayoutDimension().height : 0}
-                contentWidth={this._initComplete ? this._virtualRenderer.getLayoutDimension().width : 0}>
+                contentWidth={this._initComplete ? this._virtualRenderer.getLayoutDimension().width : 0}
+                itemContainerProps={this.props.itemContainer ? {
+                    horizontal : this.props.isHorizontal,
+                    layoutOffset : this.getCurrentScrollOffset(),
+                    layoutInterval: windowBound + this.getCurrentRenderAheadOffset(),
```

Comment:
all of need here is renderAheadOffset

Reply:
Recyclerlistview has to pass these props to scrollcomponent as only RLV will have the correct values such as that of scrollOffset